webapp/views/App/views/Analysis/Chain/Chain.js

```diff
@@ -24,10 +28,19 @@ import ButtonBar from './ButtonBar'
 
 const ChainComponent = () => {
   const analysis = useAnalysis()
```

Comment:
is the state initialized from storage (if there is) here?
should we rename analysis to state?

Reply:
no it isnt